.github/workflows/cnf-test-partner-image.yaml

```diff
@@ -56,9 +57,16 @@ jobs:
           username: ${{ secrets.QUAY_ROBOT_USERNAME }}
           password: ${{ secrets.QUAY_ROBOT_TOKEN }}
 
-      - name: 'Build and push the new `cnf-test-partner` image'
+      - name: 'Build and push the new `cnf-test-partner` latest image'
         uses: docker/build-push-action@v3
         with:
           push: true
           file: ${{ env.IMAGE_CONTAINER_FILE_PATH }}
           tags: '${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}'
```

Comment:
Why not just add the new tage here:
tags: '${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}, ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.SPECIFIC_IMAGE_TAG }} '

Reply:
Does that work?  I'll give it a try.

Reply:
Done.